readthedocs/core/models.py

```diff
@@ -1,44 +1,46 @@
-"""Models for the core app."""
+"""
+Models for the core app.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
 
-from __future__ import absolute_import
 import logging
 
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-from django.utils.translation import ugettext_lazy as _, ugettext
+from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext
+
 from annoying.fields import AutoOneToOneField
 
-STANDARD_EMAIL = "anonymous@readthedocs.org"
+STANDARD_EMAIL = 'anonymous@readthedocs.org'
 
 log = logging.getLogger(__name__)
 
 
 @python_2_unicode_compatible
-class UserProfile (models.Model):
-
-    """Additional information about a User."""
+class UserProfile(models.Model):
+    """
+    Additional information about a User.
+    """
 
-    user = AutoOneToOneField('auth.User', verbose_name=_('User'),
-                             related_name='profile')
+    user = AutoOneToOneField('auth.User', verbose_name=_('User'), related_name='profile')
     whitelisted = models.BooleanField(_('Whitelisted'), default=False)
     banned = models.BooleanField(_('Banned'), default=False)
     homepage = models.CharField(_('Homepage'), max_length=100, blank=True)
-    allow_ads = models.BooleanField(_('See paid advertising'),
-                                    help_text=_('If unchecked, you will still see community ads.'),
-                                    default=True,
-                                    )
-    allow_email = models.BooleanField(_('Allow email'),
-                                      help_text=_('Show your email on VCS '
-                                                  'contributions.'),
-                                      default=True)
+    allow_ads = models.BooleanField(
+        _('See paid advertising'),
+        help_text=_('If unchecked, you will still see community ads.'),
+        default=True,
+    )
+    allow_email = models.BooleanField(
+        _('Allow email'), help_text=_('Show your email on VCS '
+                                      'contributions.'), default=True)
```

Comment:
It got broken because the string was already splitted. I think if I remove the `'` and make the string just one thing it will be more intelligent.

Yes, it will be:

```python
    allow_email = models.BooleanField(
        _('Allow email'), help_text=_('Show your email on VCS contributions.'), default=True)
```

and, if we add a trailing comma after `default=True`, yapf will format it like this:

```python
     allow_email = models.BooleanField(
        _('Allow email'),
        help_text=_('Show your email on VCS contributions.'),
        default=True,
    )
```